Sources/BrowserServicesKit/ContentBlocking/UserScripts/SurrogatesUserScript.swift

```diff
@@ -112,18 +114,36 @@ open class SurrogatesUserScript: NSObject, UserScript {
             delegate.surrogatesUserScript(self, detectedTracker: tracker, withSurrogate: host)
         }
     }
-            
+
     private func trackerFromUrl(_ urlString: String, pageUrlString: String, _ blocked: Bool) -> DetectedTracker {
         let currentTrackerData = configuration.trackerData
         let knownTracker = currentTrackerData?.findTracker(forUrl: urlString)
         let entity = currentTrackerData?.findEntity(byName: knownTracker?.owner?.name ?? "")
         return DetectedTracker(url: urlString, knownTracker: knownTracker, entity: entity, blocked: blocked, pageUrl: pageUrlString)
     }
 
+    private static func createSurrogateFunctions(_ surrogates: String) -> String {
+        let commentlessSurrogates = surrogates.split(omittingEmptySubsequences: false, whereSeparator: \.isNewline).filter {
+            return !$0.starts(with: "#")
+        }.joined(separator: "\n")
+        let surrogateScripts = commentlessSurrogates.components(separatedBy: "\n\n")
```

Comment:
Do we know how many scripts are we expecting? Does it make sense to do a verification at least in the DEBUG configuration?

Reply:
We currently have 12: https://github.com/duckduckgo/tracker-surrogates/tree/main/surrogates I don't know if there's any plans to exponentially grow these as they're a maintenance burden and source of breakage themselves.